route/build/src/read_control.f90

```diff
@@ -403,40 +403,40 @@ SUBROUTINE read_control(ctl_fname, err, message)
  ! ---------- simulation time step, output frequency, file frequency -------
  if (masterproc) then
    write(iulog,'(2a)') new_line('a'), '---- output/simulation time steps --- '
-   write(iulog,'(A,F10.1)') '  simulation frequency:     ', dt
-   write(iulog,'(2A)')      '  history file freqeuncy:   ', trim(newFileFrequency)
-   write(iulog,'(2A)')      '  history output freqeuncy: ', trim(outputFrequency)
+   write(iulog,'(A,F10.1)') '  simulation time step <dt_qsim>:             ', dt
+   write(iulog,'(2A)')      '  history file freqeuncy <newFileFrequency>:  ', trim(newFileFrequency)
+   write(iulog,'(2A)')      '  history output freqeuncy <outputFrequency>: ', trim(outputFrequency)
  end if
 
  ! 1. Process history output frequency
  select case(trim(outputFrequency))
    case('daily', 'monthly', 'yearly') ! do nothing
    case default
-     read(outputFrequency,'(5I)',iostat=err) nOutFreq
+     read(outputFrequency,'(I5)',iostat=err) nOutFreq
      if (err/=0) then
-       message=trim(message)//'<outputFrequency> is invalid: must be daily, monthly, yearly or integer (number of time steps)'; return
+       message=trim(message)//'problem with numeric convertin - <outputFrequency>'; return
```

Comment:
It seems like the previous message was easier to understand?

Reply:
So this one is tricky.  if err is detected, it is from `read(outputFrequency,'(I5)',iostat=err) nOutFreq` and this is just converting outputFrequency (character) to nOutFreq (integer).   how about this?

'problem with converting <outputFrequency> to numeric'

Reply:
The reason I suggest that you keep the previous one is that someone might misspell one of the valid options and this is the error that will be triggered. So the previous error is better in terms of explaining to the user what they need to do. It's more correct in terms of what went wrong -- but it's better to explain to the user what they need to do to fix the problem -- rather than be more technically correct.

Reply:
put the original error message back as suggested.